src/EditorFeatures/CSharpTest/AddBraces/AddBracesTests.cs

```diff
@@ -260,13 +313,18 @@ unsafe static void Main()
             return;
         }
     }
-}");
+}",
+                (PreferBracesPreference)bracesPreference,
+                expectDiagnostic);
```

Comment:
this isn't really your fault... but this is going to take a while to review these tests.  specifically, i really want to validate all the `whenmultiline:true/false` cases.  But the PR doesn't show the code here, so i have to go to each of these and expand and validate it fits my expectations.  This will just take a bit of time. Sorry!

Reply:
The tests are split. Only one case from the original set of tests has the value true. All the new tests were added in a second commit where you can easily get context.